### ISSUE TITLE
MQTT authorization fails

### ISSUE THREAD

**Issue by snekcz** [httpyac/httpyac.github.io#58]
Hi,

I have just found this extension and it looks very interesting. Thanks for it.

However, when I try to connect to MQTT server which requires authorization it fails. 
Call:
```
MQTT tcp://localhost:8883
Topic: test/topic 
username: user
password: password
```

fails with message `Client mqttjs_c61e50e4 disconnected, not authorised.`.

When I use the same values from within a Node.js app, the connection is successful. 

Looks like the username and password are not passed to the MQTT server.

**Comment by AnWeber** [comment]
@snekcz An error has crept in. This will be corrected with the next release. Workaround in the meantime would be the following notation

```
{{
   request.options = {
      username: "user",
      password: "password"
  };
}}

MQTT tcp://localhost:8883
Topic: test/topic 
```

**Comment by snekcz** [comment]
@AnWeber Thanks a lot for the workaround and fix.

**Comment by AnWeber** [comment]
fix released with v5.8.1
